src/planwise/client/api.cljs

```diff
@@ -19,6 +19,16 @@
 
 (swap! default-interceptors (partial cons csrf-token-interceptor))
 
+;; Default json request settings
+
+(defn error-handler [{:keys [status status-text]}]
+  (.error js/console (str "Error " status " performing AJAX request: " status-text)))
+
+(def json-opts
+  {:format :json
+   :response-format :json
+   :keywords? true
+   :error-handler error-handler})
```

Comment:
We could turn `json-opts` into a function that actually receives the params, success-fn, and any other settings, and merges (and optionally preprocesses them). It could be useful to add a new `mapping-fn` that wraps the `success-fn`.